### ISSUE TITLE
Top10 Bot ?

### ISSUE THREAD

**Issue by fahadmehar** [MadaliNTCL/eggdrop-tcl#4]
Do you write !top10 TCL ? If we have seen without !top10  is not complete.

**Comment by MadaliNTCL** [comment]
There is !seen top

**Comment by fahadmehar** [comment]
Yes there is seen but you know there is !top also like !top10 chatters. 

<@Fahad> !top10
-Statistics- Top10(words): 1. Fahad(3362) 2. Zalgo(2649) 3. Statistics(1586) 4. ODay(1013) 5. CMOS(731) 6. June28(410) 7. QueenZa(365) 8. Maimu(210) 9. Deadite(182) 10. Sachin`(135)

**Comment by MadaliNTCL** [comment]
Seen script is another thing, its not a statistic script. What you are saying comes from another script.

**Comment by fahadmehar** [comment]
I know its seen script... But i am asking you to make or if you have statistic script.

**Comment by MadaliNTCL** [comment]
No i don`t have statistic script, atleast not an updated one. I will work on one in the near future.

**Comment by fahadmehar** [comment]
OK when you work on Statistics script please use ENGLISH and also stats in NOTICE
